go.mod

```diff
@@ -87,8 +87,8 @@ require (
 	golang.org/x/sync v0.1.0
 	golang.org/x/sys v0.13.0
 	golang.org/x/time v0.3.0
-	google.golang.org/genproto v0.0.0-20230306155012-7f2fa6fef1f4
-	google.golang.org/grpc v1.53.0
+	google.golang.org/genproto v0.0.0-20230410155749-daa745c078e1
+	google.golang.org/grpc v1.55.0
```

Comment:
Merge my PR then you can drop these 😉 😜 😂

Reply:
Should be dropped now :tada: